### ISSUE TITLE
SimplePagination next and previous buttons are clickable even if it's only one page

### ISSUE THREAD

**Issue by adetxt** [laravel-filament/filament#1293]
### Package

filament/tables

### Package Version

v2.9.7

### Laravel Version

v8.75

### Livewire Version

v2.9

### PHP Version

PHP 8.0

### Bug description

`SimplePagination` next and previous buttons are clickable even if it's only one page. The button is also send a request and displaying "No records found" in the next page.

### Steps to reproduce

- Create a table with only one page data, and use simple pagination
```
protected function paginateTableQuery(Builder $query): Paginator
{
      return $query->simplePaginate($this->getTableRecordsPerPage());
 }
```
- Go to the page and click next button or previous button

**Comment by ManojKiranAppathurai** [comment]
I can confirm this.

**Comment by danharrin** [comment]
I've fixed this in #1319. There is a JS bug with dynamically removing the `wire:click` from buttons, so now I have hidden them altogether when they are disabled.
